plugins/node/opentelemetry-instrumentation-mongodb/src/instrumentation.ts

```diff
@@ -417,6 +419,32 @@ export class MongoDBInstrumentation extends InstrumentationBase<
     span.setAttribute(SemanticAttributes.DB_STATEMENT, JSON.stringify(query));
   }
 
+  /**
+   * Triggers the response hook in case it is defined and enhancedDatabaseReporting is set to true.
```

Comment:
why should `enhancedDatabaseReporting` be enabled here ? Speciallly since you don't check it in the if

Reply:
You are right - at first I built it together with that dependency - fixing the documentation now.